en/camera_protocol.md

```diff
@@ -2,6 +2,31 @@
 
 The camera protocol allows to configure camera payloads and request their status. It supports photo and video cameras and includes messages to query and configure the onboard camera storage.
 
+## Camera Information
+
+### Components:
+
+ - **`MAV_COMP_ID_CAMERA`** - Component is a single camera
+[PROPOSAL]: or it is a component that controls multiple cameras.
+
+### Command interface :
+
+ - **`MAV_CMD_REQUEST_CAMERA_INFORMATION`** - Used to get cameras information. May also be used to discover available cameras using camera_id = 0. Response message is a [**`CAMERA_INFORMATION`**](http://mavlink.org/messages/common#CAMERA_INFORMATION)
```

Comment:
I'm a bit fan of "minimal style". Is it necessary for these to be bold? Possibly the first case, but not really the link.

E.g.

- **`MAV_CMD_REQUEST_CAMERA_INFORMATION`** - Used to get cameras information. May also be used to discover available cameras using camera_id = 0. Response message is a [`CAMERA_INFORMATION`](http://mavlink.org/messages/common#CAMERA_INFORMATION)

Rather than

- **`MAV_CMD_REQUEST_CAMERA_INFORMATION`** - Used to get cameras information. May also be used to discover available cameras using camera_id = 0. Response message is a [**`CAMERA_INFORMATION`**](http://mavlink.org/messages/common#CAMERA_INFORMATION)